### ISSUE TITLE
Compliance/Open Compliance Dashboard

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-compliance#57]
#### Description:

As an operator, after I have clicked on the COAM tile on the BCIERS dashboard, I am on the COAM dashboard which has tiles to the COAM dashboard and the BCCR.

Link to [Figma design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=20107-1497&t=mMCqKm8zClEjGn5g-4)

#### Acceptance Criteria:

As an operator, given that I've clicked through to the COAM dashboard, when I come to the dashboard page, I see two tiles:

- [ ] A tile called My Compliance with a description and links to:
- Compliance summaries tab
- Imposed penalties tab
- Payment summary tab

- [ ] A tile called B.C. Carbon Registry with a description and a link to [BCCR](https://www2.gov.bc.ca/gov/content/environment/climate-change/industry/offset-projects/bc-carbon-registry) [Confirm text]

When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
- Blocking #58 

#### Notes:

-

**Comment by patrickisaac** [comment]
Refinement Notes:
- Duplicate ticket - already worked on/completed. Closing ticket.
